### ISSUE TITLE
ByeBlock cannot unblock song

### ISSUE THREAD

**Issue by Taekwondodo** [jariz/byeblock#3]
Hey jariz thanks again for making the addon, really appreciate it.

Not sure if this is a known issue, but as you can see here: 
http://puu.sh/o15mo/59328fa591.jpg

I've installed the addon however as you can see "I Took a Pill in Ibiza" is still blocked for me.

Are there some songs that can't be unblocked or is it another issue?

Thanks!

**Comment by jariz** [comment]
Hey @Taekwondodo, byeblock only checks if tracks are unblocked in the states, so if a track is both blocked in your country and in the states (other examples are what so not's latest EP) it won't be able to unblock it. (hence it saying 'not available in the United States')
